DirectProgramming/C++SYCL_FPGA/Tutorials/GettingStarted/fpga_compile/README.md

```diff
@@ -333,14 +333,22 @@ For part 1 of this tutorial, only the `fpga_emu` target is available as this reg
 
 ### Read the Reports
 
-In *part2*, *part3* and *part4*, after running the `report` target, the optimization report can be viewed using the `fpga_report` application:
+In *part2*, *part3* and *part4*, after running the `report` target, the FPGA optimization report can be viewed using the Intel® oneAPI FPGA Report Main Menu:
 
+1. Open the Intel® oneAPI FPGA Report Main Menu (\<oneAPI FPGA installation directory>/share/lib/reports/html_report/report.html) in a supported browser*. If you cannot find the Intel® oneAPI FPGA Report Main Menu path, try running 'fpga_report'.
 ```
-fpga_report vector_add.report.prj/reports/vector_add_report.zip
+<browser of choice> <oneAPI installation directory>/share/lib/reports/html_report/report.html
 ```
+<img src="report_main_menu.png" height="500rem" alt='The Intel® oneAPI FPGA Report Main Menu, which includes a recently opened section and a getting started section with import file buttons.'/>
```

Comment:
The policy is to not use HTML in the READMEs (or as little as possible).
So if you can avoid it, it is preferred.

Also the images should be placed in an `assets` folder